lib/business/reservation_manager.dart

```diff
@@ -35,25 +37,31 @@ class ReservationManager extends ChangeNotifier {
 
   List<TimetableTask> get reservations => List.unmodifiable(_reservations);
   List<Boardgame> get games => List.unmodifiable(_games);
+  List<Account> get accounts => List.unmodifiable(_accounts);
   int get selectedIndex => _selectedIndex;
   int get selectedGameIndex => _selectedGame;
   int get selectedPlaceIndex => _selectedPlace;
+  int get selectedAccountIndex => _selectedAccount;
   int get selectedMode => _selectedMode;
   TimetableTask? get selectedTask =>
       selectedIndex != -1 ? _reservations[selectedIndex] : null;
```

Comment:
```suggestion
  TimetableTask? get selectedTask =>
      selectedIndex != -1 ? _reservations[_selectedIndex] : null;
```

Ha egységesítünk, legyen ez is belső változó.